SPECS/tdnf/tdnf.spec

```diff
@@ -167,9 +166,23 @@ if [[ ! -f %{_tdnf_history_db_dir}/history.db ]]; then
     %{_libdir}/tdnf/tdnf-history-util init
 fi
 
+%preun
+if  [[ $(readlink $(rpm --eval %{_bindir})/yum) == $(rpm --eval %{_bindir})/tdnf ]]; then
+  rm $(rpm --eval %{_bindir})/yum
+fi
+
 %postun
 /sbin/ldconfig
 
+%posttrans
+# Make the decision to update yum sym link during post script
+# not package build. Some workloads want to disable tdnf, so
+# decision should not be made during build but during install
+# of package.
+if [ ! -h $(rpm --eval %{_bindir})/yum ]; then
```

Comment:
i don't think this is the right thing to check; i believe we want to create the symlink iff there is no 'yum' file/link at all, otherwise we don't want to do anything. so probably this should use `-e` not `-h`.

Reply:
Ah yup, we don't want to clobber `yum` if it is a file only. So we can change the test to be for file presence (`-e`).